lib/connection/connection_config.js

```diff
@@ -483,6 +483,14 @@ function ConnectionConfig(options, validateCredentials, qaMode, clientInfo)
     disableQueryContextCache = options.disableQueryContextCache;
   }
 
+  let maxLoginTimeout = 300;
```

Comment:
Please add the new parameter to `DEFAULT_PARAMS`

Reply:
Added and changed the name to loginTimeout